src/rpc/blockchain.cpp

```diff
@@ -1754,8 +1754,10 @@ static UniValue getblockstats(const JSONRPCRequest& request)
             "}\n"
                 },
                 RPCExamples{
-                    HelpExampleCli("getblockstats", "1000 '[\"minfeerate\",\"avgfeerate\"]'")
-            + HelpExampleRpc("getblockstats", "1000 '[\"minfeerate\",\"avgfeerate\"]'")
+                    HelpExampleCli("getblockstats", R"('"00000000c937983704a73af28acdec37b049d214adbda81d7e2a3dd146f6ed09"' '["minfeerate","avgfeerate"]')") +
+                    HelpExampleCli("getblockstats", R"("1000 '["minfeerate","avgfeerate"]'")") +
```

Comment:
This doesn't equal to the original HelpExampleCli string anymore, as there are two extra quotation marks at the beginning and end:
| Branch        | Source-code representation           | Visible result  |
| ------------- |:-------------:|:-----:|
| master  | `"1000 '[\"minfeerate\",\"avgfeerate\"]'"` | `1000 '["minfeerate","avgfeerate"]'` |
| PR17831      | `R"("1000 '["minfeerate","avgfeerate"]'")"`  | `"1000 '["minfeerate","avgfeerate"]'"` |

```
$ bitcoin-cli getblockstats "1000 '["minfeerate","avgfeerate"]'"
error: Error parsing JSON:1000 '[minfeerate,avgfeerate]'
```